sci-physics/clhep/clhep-2.4.7.1-r1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="High Energy Physics C++ library"
+HOMEPAGE="
+	http://proj-clhep.web.cern.ch/proj-clhep/
+	https://gitlab.cern.ch/CLHEP/CLHEP
+"
+SRC_URI="http://proj-clhep.web.cern.ch/proj-clhep/dist1/${P}.tgz"
+
+S="${WORKDIR}/${PV}/CLHEP"
+
+LICENSE="GPL-3 LGPL-3"
+SLOT="2/${PV}"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+
+IUSE="doc static-libs test threads"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	doc? (
+		app-text/doxygen
+		dev-texlive/texlive-bibtexextra
+		dev-texlive/texlive-fontsextra
+		dev-texlive/texlive-fontutils
+		dev-texlive/texlive-latex
+		dev-texlive/texlive-latexextra
+	)
+"
+
+src_prepare() {
+	if ! use static-libs; then
+		PATCHES+=( "${FILESDIR}"/${P}-static-libs.patch )
+	fi
```

Comment:
Maybe instead of conditional epatch, you can implement a patch that causes the build system to respect an option toggle?